content/accessibility/links.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Links
+---
+
+## Overview
+Links are user interface elements that *navigate* you to a new place or new content. Contrast this with buttons, which are designed to *activate* a feature.
+
+## Why?
+Links can do things like help with page context, reference similar items of interest, and allow for endless connected information surfing through sites like Wikipedia. Links play a key part in your experience on the web, but without proper consideration they can be frustrating to use, skipped over, or completely unnoticed. 
+
+For screen reader assistive technology, links and buttons are expected to function differently from each other. If a link is activated and does not do what was expected, that can be disorienting and frustrating. 
+
+A common way a screen reader might navigate the page is by going through a list of all the links on the page. Without context, "read more" or "click here" links are not helpful. 
+
+People who have low or colorblind vision may have trouble identifying links that just use color to distinguish them from plain text, this is why keeping the underline styling on links within body text is important for identification.
+
+## How to test links
+- Check a link's function. If clicking a link activates a feature on the page, it should be a `<button>`, and vice versa. 
+- Check a link's purpose. If you can't get an idea of where a link will take you based on the link text alone, the link text should be updated. 
+- [Check a link's contrast](https://webaim.org/resources/contrastchecker/). Because links are text-based, they need to pass 4.5:1 against the background color that they are on.
+- In body text, like a Markdown file, check to make sure that all links have underline styling. 
+
+## Guidelines
+
+### For designers
+- Make sure you use the correct accent color for links in your designs. Double check the color you're using if you're using it on anything other than a white background. For more context on color, check out our [color documentation](https://primer.style/design/foundations/color).
+- Make sure a link's purpose can be understood on its own without the surrounding context in concise words. 
+- Links should always be underlined if they're in body text. 
+- Links should look like links, not buttons, except in rare circumstances, like calls to action. If the action takes you somewhere rather than doing something on the page, it should be semantically coded as a link regardless of the visual style.
+
+<DoDontContainer stacked>
```

Comment:
The Do/Don't area was not visually clear to me. Could we leverage the [Doctocat content template](https://www.figma.com/file/YQT3nlmvSdZRPBnkDXvTFk/?node-id=0%3A1) (see below), to create more clearly distinguished guidelines? 
![Stacked template guidance](https://user-images.githubusercontent.com/7228610/160145884-0239d617-d76a-4a5d-b3a8-2b56beafedbd.png)